TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/api/implementations/objects/PlayerObjectBasicImplementation.java

```diff
@@ -87,7 +90,9 @@ public boolean isOnline() {
 
     @Override
     public APIRequestFuture<Void> sendToServer(ServerObject serverObject) {
```

Comment:
We could return an APIRequestFuture<Boolean>, stating whether the player has been sent to the server successfully.
Add the corresponding JavaDoc above the method:

```
/**
 * @return An APIRequestFuture<Boolean> specifying if the player has been sent to the server successfully
*/
```